G1ANT.Addon.Net/SimplifiedMessageSummary.cs

```diff
@@ -1,27 +1,321 @@
 ﻿using MailKit;
 using MimeKit;
-using G1ANT.Language;
+using System;
+using System.IO;
+using System.Linq;
+using System.Collections.Generic;
+using G1ANT.Addon.Net.Models;
 
 namespace G1ANT.Addon.Net
 {
-    public class SimplifiedMessageSummary : MessageSummary
```

Comment:
So great that you removed this inheritance, because it looks it should be done in the opposite direction.
👍